test

```diff
@@ -98,7 +98,6 @@ function run_isolated_tests()
   cd "${path_test}"
 
   ws helm kubeval qa
-  rm -rf .my127ws/helm/qa/charts/ .my127ws/helm/qa/requirements.lock
```

Comment:
I find it useful from a dev perspective keeping it, as I need to run it more than once